src/gen_cranberry/gc_dev_hih6131.cpp

```diff
@@ -4,21 +4,21 @@
 static HIH613x hih6131(0x27);
 
 void gc_dev_hih6131_open(void){
-    Wire.begin(9600);
+
 }
 
-int gc_dev_hih6131_temp_decic_read(void){
+uint16_t gc_dev_hih6131_temp_decik_read(void){
     int16_t value = 89;
 
     #ifndef SEN_STUB
     hih6131.update();
-    value = hih6131.getTemperature();
+    value = hih6131.getTemperature() + 273.15;
```

Comment:
It looks like we're dropping precision here: the decimal points will get cut off in the conversion to a int16_t. Is it really returning deci-kelvin?